aws/resource_aws_instance.go

```diff
@@ -433,6 +433,22 @@ func resourceAwsInstance() *schema.Resource {
 					},
 				},
 			},
+
+			"credit_specification": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"cpu_credits": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
```

Comment:
Should this have validation? I'm not familiar with the API, but I only seen `unlimited` and `standard` in the results? Is it really free form?

Reply:
I believe the API complains if you use something other than `unlimited` and `standard`, leaving this opaque in terraform gives _out of the box_ support for new `cpu_credit` types in the future.